Firmware/ultralcd.cpp

```diff
@@ -1992,22 +2067,44 @@ static void lcd_menu_temperatures()
 #define VOLT_DIV_R2 2370
 #define VOLT_DIV_FAC ((float)VOLT_DIV_R2 / (VOLT_DIV_R2 + VOLT_DIV_R1))
 #define VOLT_DIV_REF 5
+//! @brief Show Voltages
+//!
+//! @code{.unparsed}
+//! |01234567890123456789|
+//! |                    |
+//! | PWR:         00.0V |	c=12 r=1
+//! | Bed:         00.0V |	c=12 r=1
+//! |                    |
+//! ----------------------
+//! @endcode
+//! @todo Positioning of the messages and values on LCD aren't fixed to their exact place. This causes issues with translations.
 static void lcd_menu_voltages()
 {
 	lcd_timeoutToStatus.stop(); //infinite timeout
 	float volt_pwr = VOLT_DIV_REF * ((float)current_voltage_raw_pwr / (1023 * OVERSAMPLENR)) / VOLT_DIV_FAC;
 	float volt_bed = VOLT_DIV_REF * ((float)current_voltage_raw_bed / (1023 * OVERSAMPLENR)) / VOLT_DIV_FAC;
 	lcd_home();
-	lcd_printf_P(PSTR(" PWR:      %d.%01dV\n" " BED:      %d.%01dV"), (int)volt_pwr, (int)(10*fabs(volt_pwr - (int)volt_pwr)), (int)volt_bed, (int)(10*fabs(volt_bed - (int)volt_bed)));
+	lcd_printf_P(PSTR(" $S:      %d.%01dV\n" " $S:      %d.%01dV"), _i("PWR"), (int)volt_pwr, (int)(10*fabs(volt_pwr - (int)volt_pwr)), _i("Bed"), (int)volt_bed, (int)(10*fabs(volt_bed - (int)volt_bed)));  ////c=12 r=1  ////c=12 r=1
```

Comment:
c=12 is not correct in this case - it should be c=3 because this hard-coded view expects exactly 3 characters for BED and PWR. Anyway, there is a plenty of space to enable for longer labels in other languages. Also, the printf can be made to justify a string left and pad it with spaces by using %-12.12S. However, the strings must include the colon (i.e. BED:), I still haven't found any trick how to justify without the ':'

Reply:
Just remembered that i already solved it in another fork by using:
`	lcd_printf_P(PSTR(ESC_H(1,1) "%S:" ESC_H(14,1) "%d.%01dV" ESC_H(1,2) "%S:" ESC_H(14,2) "%d.%01dV"), _i("PWR"), (int)volt_pwr, (int)(10*fabs(volt_pwr - (int)volt_pwr)), _i("BED"), (int)volt_bed, (int)(10*fabs(volt_bed - (int)volt_bed)));
 `